lib/sidekiq/dynamic_queues/attributes.rb

```diff
@@ -109,7 +122,24 @@ def expand_queues(queues)
           end
         end
 
-        return matched_queues.collect { |q| "queue:#{q}" }.uniq
+        return matched_queues.sort
+      end
+
+      def expand_queues_for_display(queues)
```

Comment:
This is for the dashboard UI, where it is important to see what would be processed.